### PR TITLE
Bug fixes and Deployment Improvement

### PR DESCRIPTION
**Pull request by sauban**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,10 +28,10 @@ install:
 - npm install
 - docker build -t cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER .
 - if [ "$TRAVIS_BRANCH" == "release" ]; then
-    docker build -t cloudboost/cloudboost:latest .;
+    docker tag cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER cloudboost/cloudboost:latest;
   fi
 - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker build -t cloudboost/cloudboost:staging .;
+    docker tag cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER-master;
```
Comment (nawazdhandala) [inline .travis.yml]: Just push to :master and do not have versioning in this case.

Comment (sauban) [reply .travis.yml]: Oh okay, so just `cloudboost/cloudboost:master` then.

Comment (nawazdhandala) [reply .travis.yml]: Yes.

File: .travis.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ install:
     docker tag cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER cloudboost/cloudboost:latest;
   fi
 - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker tag cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER-master;
+    docker tag cloudboost/cloudboost:2.0.$TRAVIS_BUILD_NUMBER cloudboost/cloudboost:master;
```
Comment (nawazdhandala) [inline .travis.yml]: Do not tag with build number when on master.

**Pull request merged**